.changeset/pretty-owls-pay.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin-theme": patch
```

Comment:
I think this should be at least "minor", as this adds new styling and is not just a bugfix.